Gems/ROS2/Code/Source/RobotImporter/RobotImporterWidget.cpp

```diff
@@ -0,0 +1,318 @@
+                AZ_Printf("Wizard", "Wizard skips m_checkUrdfPage since there is no errors in URDF");
+                m_meshNames = Utils::GetMeshesFilenames(m_parsedUrdf->getRoot(), true, true);
+            }
+            else
+            {
+                report += "# " + tr("The URDF was not opened") + "\n";
+                report += tr("URDF parser returned following errors:") + "\n\n";
+            }
+            if (!log.empty())
+            {
+                report += "`";
+                report += QString::fromUtf8(log.data(), int(log.size()));
+                report += "`";
+            }
+            m_checkUrdfPage->ReportURDFResult(report, m_parsedUrdf != nullptr);
+        }
+    }
+
+    void RobotImporterWidget::onCurrentIdChanged(int id)
+    {
+        AZ_Printf("Wizard", "Wizard at page %d", id);
+
+        if (currentPage() == m_assetPage)
+        {
+            FillAssetPage();
+        }
+        else if (currentPage() == m_prefabMakerPage)
+        {
+            FillPrefabMakerPage();
+        }
+    }
+
+    void RobotImporterWidget::FillAssetPage()
+    {
+        m_assetPage->ClearAssetsList();
+        if (m_parsedUrdf)
+        {
+            m_urdfAssetsMapping = AZStd::make_shared<Utils::UrdfAssetMap>(Utils::FindAssetsForUrdf(m_meshNames, m_urdfPath));
+            auto colliders_names = Utils::GetMeshesFilenames(m_parsedUrdf->getRoot(), false, true);
+            auto visual_names = Utils::GetMeshesFilenames(m_parsedUrdf->getRoot(), true, false);
+            for (const AZStd::string& mesh_path : m_meshNames)
+            {
+                const QString mesh_pathqs = QString::fromUtf8(mesh_path.data(), mesh_path.size());
+                const QString kNotFound = tr("not found");
+
+                if (m_urdfAssetsMapping->contains(mesh_path))
+                {
+                    QString type = kNotFound;
+                    QString source_path = kNotFound;
+                    auto crc = AZ::Crc32();
+                    QString tooltip = kNotFound;
+                    bool visual = visual_names.contains(mesh_path);
+                    bool collider = colliders_names.contains(mesh_path);
+                    if (visual && collider)
+                    {
+                        type = tr("Visual and Collider");
+                    }
+                    else if (visual)
+                    {
+                        type = tr("Visual");
+                    }
+                    else if (visual)
+                    {
+                        type = tr("Collider");
+                    }
+
+                    if (m_urdfAssetsMapping->contains(mesh_path))
+                    {
+                        const auto& asset = m_urdfAssetsMapping->at(mesh_path);
+                        const AZStd::string& product_path = asset.m_availableAssetInfo.m_productAssetRelativePath;
+                        const AZStd::string& resolved_path = asset.m_resolvedUrdfPath.data();
```

Comment:
The product_path and resolved_path variables should be [lowerCamelCase](https://github.com/o3de/sig-core/blob/main/governance/Coding-Standards-and-Style-Guide.md#variable-names).
```suggestion
                        const AZStd::string& productPath = asset.m_availableAssetInfo.m_productAssetRelativePath;
                        const AZStd::string& resolvedPath = asset.m_resolvedUrdfPath.data();
```